### PR TITLE
added timestamp formatting

### PR DESCRIPTION
**Pull request by PLSV7Durianpay**

### CODE REVIEW

File: woo-durianpay/woo-durianpay.php [hunk 1]
```diff
@@ -737,14 +737,31 @@ private function getOrderCreationData($orderId)
...
 
             return $data;
         }
 
+        private function getExpiryTimestamp($expiryRequest) {
+            # expiry request is in the format hours:minutes:seconds
+            # hours can have a value of more than 24, meaning you can add 1 or more days for an expiry date
+            # minutes can have a value of more than 60, meaning you can add 1 or more hours for an expiry date
+            # likewise for seconds, a value of more than 60 means we are adding 1 or more minutes for an expiry date
+            # example => 100:100:100 100 hours, 100 minutes and 100 seconds meaning order will expire 4 days, 5 hours, 41 minutes and 40 seconds from now
```
Comment (supriyadi687) [inline woo-durianpay/woo-durianpay.php:760]: Any input field from woocommerce admin page? If so please add the hint or description on how to use this field @PLSV7Durianpay 

cc @akhamat-durianmoney

**Review by supriyadi687 — COMMENTED** [review]
Lgtm. Please make sure the woocommerce developer or admin from merchant side understood how to use the expiry timestamp. @PLSV7Durianpay 

Cc @akhamat

File: woo-durianpay/woo-durianpay.php (hunk 1 above)
Comment (PLSV7Durianpay) [reply woo-durianpay/woo-durianpay.php:760]: I have tested this with @arifbudurian and I have ensured that the order expiry methods work as planned

**Review by arifbudurian — APPROVED** [review]
LGTM

**Pull request merged**